paasta_tools/cli/cmds/status.py

```diff
@@ -158,7 +158,8 @@ def report_status_for_cluster(service, cluster, deploy_pipeline, actual_deployme
 
     if len(deployed_instances) > 0:
         status = execute_paasta_serviceinit_on_remote_master('status', cluster, service, ','.join(deployed_instances),
-                                                             system_paasta_config, stream=True, verbose=verbose)
+                                                             system_paasta_config, stream=True, verbose=verbose,
+                                                             ignore_ssh_output=True)
```

Comment:
So that's one of the reasons why I added this flag here, for stuff like `status`, it isn't critical if an ssh connection fails, and even if it does this will toss up an exception because the return code won't be 0